R/Phenotyping.R

```diff
@@ -430,5 +432,13 @@ MakePhenotypingDotPlot <- function(seuratObj,
     xlab("Genes") +
     labs(color=colorLabel, size = "Percentage of Cells\nWith Gene Expression")
   
+  scaling <- if(scale.by != "size" | scale.by != "radius"){
```

Comment:
being picky, but it's probably easier to structure like:

if (size == 'x') {

} else if (size == 'y') {

} else {
    stop('.....')
}

the point being that you test the allowable values, and then die if it's not one of them. Easier to maintain than constantly adding to:
```
if(scale.by != "size" | scale.by != "radius"){
```

Reply:
Agreed and changed